src/content/api/cli.md

```diff
@@ -122,63 +175,55 @@ __Print result of webpack as a JSON__
 
 ```bash
 webpack --json
-webpack --json > stats.json
+webpack --json stats.json
```

Comment:
Maybe add a note here `If you want to store stats as json instead of printing it, you can use -`

Reply:
Yeah sounds good